### ISSUE TITLE
Errors with v0.8.0b7 and v0.7.1?

### ISSUE THREAD

**Issue by 3d73a4a412** [ActivityWatch/activitywatch#255]
I just tried installing AW on my win10 machine and got two different errors with two different versions.
v0.7.1 gives me an error:
"Windows cannot access the specified device path, or file. You may not have the appropriate permissions to access the item"
I also tried to open it as admin but still same error.

and v0.8.0b7 gives me the error:
Failed to execute script pyi_rth_qt5 
I know nothing about python but tried finding installing pyqt but that didnt seem to help.

I admittedly havent tried compiling it but that is a bit daunting for a non-coder such as myself.

**Comment by johan-bjareholt** [comment]
Beta 7 is broken on windows, try beta 6!

As seen in the release notes:

>  Doesn't work on Windows, please use v0.8.0b6 until we fix it.

Would make more sense if we removed it completely from the downloads though, it's very misleading.

**Comment by 3d73a4a412** [comment]
Ah, sorry i didnt see that!
I just took a look at the page, it seems all the v.6 links are for source? Perhaps I am looking in the wrong place? Is there a page with compiled binaries of AW?
Thank you!!!

**Comment by johan-bjareholt** [comment]
Hm that's odd, 0.8 beta 6 works for me on windows.
What happens if you just for example start aw-server.exe, aw-watcher-afk.exe or aw-watcher-window.exe directly, do you get any log there?

**Comment by 3d73a4a412** [comment]
They each start then apparently stop. I pulled up the task manager and processes so I could see a bit better what was happening and each time I clicked on them they started up they would show up for a second as would something like conhost.exe then both would "disappear", presumably crashing. I was only able to start each of them once, after that I was not able to start them again, I would dbl click on them and nothing, they wouldnt even come up in the processes window.
